collections/collections.js

```diff
@@ -352,6 +352,10 @@ Schema.ContextBlocks = new SimpleSchema({
   addedAt: {
     type: Date
   },
+  videoMarker: {
+    type: String,
```

Comment:
Ah, I see what you're getting at. Javascript's `Number` should have more than enough precision for what you're doing here though. `Number` is also more straightforward and takes less code to use, and has other advantages, like if you ever wanted to sort by it, or filter by a range, when doing a server-side lookup in Mongo.